archivist_samples/software_bill_of_materials/sbom_advanced_demo.py

```diff
@@ -0,0 +1,168 @@
+#   Copyright 2021 Jitsuin, inc
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+#   This is API SAMPLE CODE, not for production use.
+
+# pylint:  disable=missing-docstring
+
+from archivist import archivist
+from software_package import SoftwarePackage
```

Comment:
where is softwarepackage?

this import will not work is this forms part of a library package...

Reply:
Fixed in update 5 (wheel version)